api/openapi-spec/swagger.json

```diff
@@ -786,7 +786,11 @@
           "description": "Rolling update config params. Present only if type = \"RollingUpdate\"."
         },
         "type": {
-          "description": "Type of daemon set update. Can be \"RollingUpdate\" or \"OnDelete\". Default is RollingUpdate.",
+          "description": "Type of daemon set update. Can be \"RollingUpdate\" or \"OnDelete\". Default is RollingUpdate.\n\nPossible enum values:\n - `\"OnDelete\"` Replace the old daemons only when it's killed\n - `\"RollingUpdate\"` Replace the old daemons by new ones using rolling update i.e replace them on each node one after the other.",
```

Comment:
I know it's painful, but would you be able to go through these comments to make them consistently use double-quotes within backquotes, like this:
```suggestion
          "description": "Type of daemon set update. Can be `\"RollingUpdate\"` or `\"OnDelete\"`. Default is `"RollingUpdate"`.\n\nPossible enum values:\n - `\"OnDelete\"` Replace the old daemons only when it's killed\n - `\"RollingUpdate\"` Replace the old daemons by new ones using rolling update i.e replace them on each node one after the other.",
```

Reply:
OK, let's file an issue and solve this after code thaw. Merging this is going to be incredibly difficult before code-freeze or shortly after code thaw.

Reply:
We will need to change comments of numerous types around our APIs because how common quoting a constant is. Updating the comments of each type is out of scope of this PR, which mainly focus on schema generation.

Reply:
Yeah, we should probably add something to the API conventions also if we don't have it.

Reply:
Yeah, we should probably add something to the API conventions also if we don't have it.

Reply:
Sent an issue to track the progress of making quotes consistent across our APIs.

Reply:
Thanks - an unambiguous, documented convention and a plan to converge is what I want.  In general the API comments are not expected to be markdown, so we will discuss what we want that convention to be. :)